source/server/options_impl.h

```diff
@@ -32,30 +32,65 @@ class OptionsImpl : public Server::Options {
 
   // Server::Options
   uint64_t baseId() const override { return base_id_; }
+  void setBaseId(uint64_t base_id) { base_id_ = base_id; };
```

Comment:
super nit: since these aren't actually part of the interface, can you move the new methods above the `Server::Options` comment?